VitalTrackApp/src/screens/Login.js

```diff
@@ -1,77 +1,47 @@
-import auth from '@react-native-firebase/auth';
-import {useNavigation} from '@react-navigation/native';
-import React, {useState} from 'react';
 import {
-  Image,
-  StyleSheet,
+  View,
   Text,
   TextInput,
   TouchableOpacity,
-  View,
+  StyleSheet,
+  Image,
 } from 'react-native';
-
+import { useNavigation } from '@react-navigation/native'; // Import useNavigation
+// test
 export default function LoginScreen() {
-  const [email, setEmail] = useState('');
-  const [password, setPassword] = useState('');
-  const navigation = useNavigation();
-
-  function handleSubmit() {
-    loginUser(email, password);
-  }
-
-  async function loginUser(email, password) {
-    try {
-      const userCredential = await auth().signInWithEmailAndPassword(
-        email,
-        password,
-      );
-      const user = userCredential.user;
-      console.log('User signed in!', user);
-
-      if (navigation) {
-        navigation.navigate('Home');
-      } else {
-        console.log('Navigation object is undefined');
-      }
-    } catch (error) {
-      console.error('Error during sign in:', error.message);
-    }
-  }
+  const navigation = useNavigation(); // Access navigation
 
   return (
-    <View style={styles.container}>
-      <Image
-        source={require('../../assets/images/VitalTrack-Logo.png')}
-        style={styles.logo}
-      />
+    <View style={styles.container}> 
+      <Image source={require('../../assets/images/VitalTrack-Logo.png')} style={styles.logo} />
 
       <Text style={styles.title}>Welcome</Text>
 
       <TextInput
-        placeholder="Email"
+        placeholder="Username"
```

Comment:
why are we reverting this to username? shouldnt this be Email.

Reply:
Good catch, I didnt edit those files but im thinking something went wrong when I merged with main